packages/dataviews/src/layouts/grid/style.scss

```diff
@@ -71,6 +71,14 @@
 			align-items: flex-start;
 			min-height: $grid-unit-30;
 
+			&:has(.dataviews-view-grid__field-value:not(:empty)) {
+				display: flex;
```

Comment:
Is this necessary? It doesn't seem to get applied, and I noticed no difference after commenting it out to test.

Reply:
It is being applied. I have added it for better readability, even though it will work without it. If you prefer, I can remove it.
<img width="503" alt="image (1)" src="https://github.com/user-attachments/assets/3be7ef67-e07a-4f00-acfa-2522629d22df">

Reply:
Ah yes, I see. However, `display: flex` is already applied by the containing `Flex` component, so redeclaring it here doesn't seem to serve a purpose?

Reply:
Okay, I have removed it.